### PR TITLE
Replace various config files with a single config

### PR DESCRIPTION
**Pull request by bsweger**
This is the re-factor for [DB-296](https://federal-spending-transparency.atlassian.net/browse/DB-296).

* Get all config settings via import from dataactcore
* Remove app init scripts that prompt for config settings
* Interfaces now get db names from config

There will be a subsequent PR that includes the corresponding doc changes.

### CODE REVIEW

**Comment by bsweger** [comment]
@jworcestBAH or @nmonga91 ready for review

File: dataactvalidator/scripts/setupValidationDB.py [hunk 1]
```diff
@@ -1,71 +1,63 @@
...
-    if(hardReset) :
-        runCommands(ValidatorValidationInterface.getCredDict(),hardResetSQL,"validation")
-    runCommands(ValidatorValidationInterface.getCredDict(),sql,"validation")
+from dataactcore.scripts.databaseSetup import createDatabase
+from dataactcore.config import CONFIG_DB
+from dataactvalidator.models import validationModels
+from dataactvalidator.models.validationModels import FileType, RuleType, FieldType, MultiFieldRuleType
+from dataactvalidator.interfaces.validatorValidationInterface import ValidatorValidationInterface
+
+def setupValidationDB(hardReset = False):
```
Comment (nmonga91) [inline dataactvalidator/scripts/setupValidationDB.py]: When running with hardReset = false, truncated tables are not repopulated.

**Comment by nmonga91** [comment]
No other concerns found. Merging into dev.

File: dataactvalidator/scripts/setupValidationDB.py (hunk 1 above)
Comment (bsweger) [inline dataactvalidator/scripts/setupValidationDB.py]: @nmonga91  Pushed a fix for this

**Pull request merged**